src/absil/ilread.fs

```diff
@@ -2222,8 +2222,6 @@ and seekReadMethodDefAsMethodData ctxt idx =
    ctxt.seekReadMethodDefAsMethodData idx
 and seekReadMethodDefAsMethodDataUncached ctxtH idx =
    let ctxt = getHole ctxtH
-   let (_code_rva, _implflags, _flags, nameIdx, typeIdx, _paramIdx) = seekReadMethodRow ctxt idx
-   let nm = readStringHeap ctxt nameIdx
```

Comment:
Why was this code moved in position? I don't think it matters, but best to keep it as it was if the change isn't necessary for the fix